exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -199,7 +203,8 @@ void createTestKitWithSeveralValidators() {
   @Test
   void createTestKitWithAuditorAndAdditionalValidators() {
     short validatorCount = 2;
-    try (TestKit testKit = TestKit.builder(EmulatedNodeType.AUDITOR)
+    try (TestKit testKit = TestKit.builder()
+        .withNodeType(EmulatedNodeType.AUDITOR)
```

Comment:
Shall we use `with` prefix, or maybe drop it (e.g., `nodeType`) or add usual `set` and `add` depending on the operation? On the other hand, if used consistently, it slightly helps with discoverability (if **all** use `with`, you can type `wi` and see what it could do).

Reply:
I like having same prefix for the reason you stated - it's nice to have all methods starting with `with`